modules/programs/neomutt-accounts.nix

```
@@ -21,6 +21,27 @@ let
   };
 
 in {
+  options.notmuch.neomutt = {
+    enable = mkEnableOption "Use Notmuch in NeoMutt" // {
```

```suggestion
    enable = mkEnableOption "Notmuch support in NeoMutt" // {
```
The description is expanded to "Whether to enable ...".